schema.sql

```diff
@@ -4,4 +4,16 @@
 CREATE TABLE animals (id integer, name text, date_of_birth date, escape_attempts integer, neutered boolean, weight_kg decimal) ;
 ALTER TABLE animals ADD species varchar(255);
 
+CREATE table owners (id SERIAL PRIMARY KEY, full_name VARCHAR(100), age INT);
+CREATE TABLE species (id SERIAL PRIMARY KEY, name VARCHAR(255));
+
+ALTER TABLE animals ADD COLUMN new_id SERIAL PRIMARY KEY;
+ALTER TABLE animal DROP COLUMN id;
+ALTER TABLE animals RENAME COLUMN new_id TO id;
+SELECT * FROM animals;
+
+ALTER TABLE ANIMALS DROP COLUMN SPECIES;
+ALTER TABLE ANIMALS ADD COLUMN species_id INT REFERENCES species(id);
```

Comment:
- [ ] Good job on updating the `animals` table with the `owner_id` column and the corresponding insert queries. However, you also need to add insert queries for the `species_id` column. Please ensure that you follow the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Modify%20your%20inserted,animals%20are%20Pokemon) precisely and update the code accordingly. 
- [ ] Once you've made these updates, remember to verify that everything works as expected and, if necessary, update the screenshots in the PR Description to accurately reflect the current state of your project. 📊📝🐾